addons/controllers/testdata/test-kapp-controller-2.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: test-kapp-controller-2
+  namespace: default
+  annotations:
+    tkg.tanzu.vmware.com/tkg-http-proxy: "foo.com"
+    tkg.tanzu.vmware.com/tkg-https-proxy: "bar.com"
+    tkg.tanzu.vmware.com/tkg-no-proxy: "foobar.com"
+    tkg.tanzu.vmware.com/tkg-proxy-ca-cert: "dummyCertificate"
+    tkg.tanzu.vmware.com/tkg-ip-family: "ipv4"
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+        - 100.96.0.0/11
+    services:
+      cidrBlocks:
+        - 100.64.0.0/13
+---
+apiVersion: run.tanzu.vmware.com/v1alpha3
+kind: KappControllerConfig
+annotations:
+  tkg.tanzu.vmware.com/addon-paused: ""
```

Comment:
To reviewer: `tkg.tanzu.vmware.com/addon-paused` annotation added to pause reconciliation